showcase/src/app/app.component.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* tslint:disable */
```

Comment:
Let's avoid disabling tslint completely because then all rules will be disabled for this file.
So we should just disable those rules that are not needed/relevant in this case.
For example: `/* tslint:disable:rule1 rule2 ruleN */`